docs/source/Support/bskReleaseNotes.rst

```diff
@@ -42,6 +42,8 @@ Version |release|
   outside of the release cycle.
 - updated plotting of ``opNav`` example scenarios to work again with latest version of ``matplotlib``
 - fixed a slew of compiler warnings when compiling with Xcode 15
+- Removed the :ref:`prescribedMotionMsgPayload` message and replaced with two separate
```

Comment:
his should probably read "Deprecated the ..."

Reply:
Done